activity-tracking/routes/goals.js

```diff
@@ -18,14 +18,16 @@ router.post('/add', async (req, res) => {
   console.log(req.body)
   try {
 
-    const { username, exerciseType, goalType, goalUnit, goalValue } = req.body;
+    const { username, exerciseType, goalType, goalUnit, goalValue, caloriesGoal, goalAim } = req.body;
 
     const newGoal = new Goal({
       username,
       exerciseType,
       goalType,
       goalUnit,
-      goalValue: Number(goalValue)
+      goalValue: Number(goalValue),
+      caloriesGoal: Number(caloriesGoal),
```

Comment:
what is the difference between goalValue and caloriesGoal please?

Reply:
goalValue is the amount associated with the weekly exercise goal. Calories goal, is a value for the daily calories goal that the user can set as well.